sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,907 @@
+# Preamble
+
+SIP Number: 022
+
+Title: A Trustless 2-way Bitcoin Peg for Stacks
```

Comment:
```suggestion
Title: A Trust-minimized 2-way Bitcoin Peg for Stacks
```

The term "trust-minimized" seems to be preferred for describing the trust model in a succinct way, as discussed [here](https://twitter.com/muneeb/status/1712884857523638459).

cc @muneeb-ali